### ISSUE TITLE
Accessibility Issue: <ibm-search>

### ISSUE THREAD

**Issue by amchris98** [IBM/carbon-components-angular#1739]
## Detailed description

> Describe in detail the issue you're having. 
> 
If you use two ibm search components on the same page, you get an accessibility violation from IBM's accessibility plugin about "Multiple elements with "search" role do not have unique labels" and the only fixes are to add the aria-label or aria-labeledby attributes to the elements with the "search" role.
Edit: issue also happens if you have one `<ibm-search>` and one `<ibm-table-toolbar-search>`
> Is this a feature request (new component, new icon), a bug, or a general issue?
>
General Issue
> Is this issue related to a specific component?
>
Search component, `<ibm-search>`
> What did you expect to happen? What happened instead? What would you like to see changed?
>
Arialabel input needs to be added and binded to the [div with the 'search' role](https://github.com/IBM/carbon-components-angular/blob/a4953f2de3b2b3b37541978ea65cb589313005cd/src/search/search.component.html#L1-L14)
> What browser are you working in?
>
Chrome
> What version of the Carbon Design System are you using?
>
v4
>  What offering/product do you work on? Any pressing ship or release dates we should be aware of?
>
MCMP
## Steps to reproduce the issue
1. Install the [IBM accessibility checker](https://www.ibm.com/able/toolkit/tools/)
2. Run it on a webpage with two <ibm-search> components or one `<ibm-search>` and one `<ibm-table-toolbar-search>
`

**Comment by zvonimirfras** [comment]
You can already change the label via `label` input. That should be enough since label has text and is connected to input via `for` and `id`, you shouldn't need aria-label and aria-labelledby.
Have you tried changing it?

**Comment by amchris98** [comment]
Yea, the checker still throws an error 
![image](https://user-images.githubusercontent.com/34778295/109048290-8b5c7480-769c-11eb-8b9b-f03b5f85300e.png). Also, I made a mistake in the original post, using an `ibm-search` and `ibm-table-toolbar-search` in our project, so an input would have to be added to that component as well unless there's another solution I'm not aware of.

**Comment by zvonimirfras** [comment]
Looks like [we'll need an ariaLabel input](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Search_role) after all.

It should be fairly straightforward. Is this something you'd know how to add and be interested in contributing?

**Comment by amchris98** [comment]
Sure I can try, never contributed to an open source before

**Comment by zvonimirfras** [comment]
Welcome to open source! 🥳 
If you have questions about the process that you can't find answer to in our docs, feel free to reach out 👍
